tao/tao_bridge/passes/tao_clone_constants_for_better_clustering.cc

```diff
@@ -93,6 +93,27 @@ StatusOr<bool> IsSmallHostConstant(Node* n) {
   return total_elements < kSmallTensorThreshold;
 }
 
+// almost same with IsSmallHostConstant but skip the check of op's size
+// for fakequant's scale and zero_point
+StatusOr<bool> IsFakeQuantArgumentConstant(Node* n) {
+  if (!n->IsConstant()) {
+    return false;
+  }
+  DeviceNameUtils::ParsedName parsed;
+  TF_RET_CHECK(
+      DeviceNameUtils::ParseFullName(n->assigned_device_name(), &parsed));
+  if (parsed.type != DEVICE_CPU) {
+    return false;
+  }
+
+  std::string nodeName = n->name();
+  if (nodeName.find("fake_quantize") != std::string::npos &&
```

Comment:
Is this robust enough? Only check by node name?

Reply:
this is a prototype patch right now, will add more checks later.